dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -116,7 +134,9 @@ public void testDashboard()
       Nodes href = document.query("//tr/td[@class='artifact-name']/a/@href");
       for (int i = 0; i < href.size(); i++) {
```

Comment:
I thought this loop could use `toList` function to eliminate the index variable `i`, but it turned out `artifacts.get(i)` uses the index variable.